src/ui/ExportUIFunctions.jl

```diff
@@ -70,6 +70,7 @@ function setup_blink_window(; darkmode=true, frame=true, dev_tools=false, show_w
 
     #Update GUI's home
     body!(w, *(sidebar, index); async=false) #async false is important to set the background correctly
+    sleep(5)
```

Comment:
If this delay fixes the tests, let's put it as an optional kwarg, `sleep_after_window=0` and only set it up as `5` for the tests.